.github/workflows/backup-cognito-users.yaml

```diff
@@ -41,18 +41,19 @@ jobs:
       - id: save-to-s3
         name: save backup json to s3
         run: |-
+          BUCKET_NAME="biomage-backups-production-${{steps.setup-aws.outputs.aws-account-id}}"
           current_date=$(date '+%Y-%m-%d.%H-%M-%S')
```

Comment:
I found out that our cognito backups were broken because of the bucket name change